pkg/api/api.go

```diff
@@ -380,6 +380,7 @@ func (hs *HTTPServer) registerRoutes() {
 		apiRoute.Get("/search/", routing.Wrap(hs.Search))
 
 		// metrics
+		// Deprecated: use /ds/query API instead.
```

Comment:
Not sure this makes sense because no one would ever call this as a function/method and the `hs.QueryMetrics` already includes the deprecation notice.

Reply:
Yeah maybe you're right - I just figured it's another place people scan for what endpoints to use. This way they they will know without having to dig deeper

Reply:
Make sense.